protocols/luci-proto-openfortivpn/root/usr/libexec/rpcd/luci.openfortivpn

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env lua
+
+local json = require "luci.jsonc"
+local fs   = require "nixio.fs"
+
+local function readfile(path)
+	if fs.stat(path, "type") == "reg" then
+		local s = fs.readfile(path)
+		return s and (s:gsub("^%s+", ""):gsub("%s+$", ""))
+	else
+		return null
+	end
+end
+
+local function writefile(path, data)
+	local n = fs.writefile(path, data)
+	return (n == #data)
+end
+
+local function parseInput()
+	local parse = json.new()
+	local done, err
+
+	while true do
+		local chunk = io.read(4096)
+		if not chunk then
+			break
+		elseif not done and not err then
+			done, err = parse:parse(chunk)
+		end
+	end
+
+	if not done then
+		print(json.stringify({ error = err or "Incomplete input" }))
+		os.exit(1)
+	end
+
+	return parse:get()
+end
+
+if arg[1] == "list" then
+	print(json.stringify({
+		getCertificates = {
+			interface = "interface"
+		},
+		setCertificates = {
+			interface = "interface",
+			user_cert = "user-cert",
+			user_key = "user-key",
+			ca_file = "ca-file"
```

Comment:
The rpcd variables are not yet correct! https://github.com/openwrt/luci/pull/5448/commits/b1d8328252137906bc1f04e6f58fcce210eea129#diff-edf2506e6803ceae7afb8f2915c5a7d056c9d21020346f52248602ef3382fa31R17

I have now printed out everything again with `ubus -v list` and it seems as if `_` is usual in Openwrt for separating the variables name. So I would suggest that we use this separator. So we have **user_cert**, **user_key** and **ca_file**.
I know I said we should use `-'. Sorry for the inconvenience.

Reply:
Have made this change. Thanks!